Yona/Yona/Others/EnvironmentManager.m

```diff
@@ -25,33 +25,30 @@ @implementation EnvironmentManager
 + (NSString *)baseUrlString
 {
     NSString *url = [[NSUserDefaults standardUserDefaults] stringForKey:@"YONA_URL"];
-    if (url == nil){
-//        url = @"https://beta.prd.yona.nu/";
-       // url = @"https://app.prd.yona.nu/";
-        url = @"https://app.beta.yona.nu/";
-//        url = @"http://mobtest.dev.yona.nu/";
+    if (url == nil || [url isEqual:@""]){
+        url = @"https://app.prd.yona.nu/"; // Production Enviroment
         [[NSUserDefaults standardUserDefaults] setObject:url forKey:@"YONA_URL"];
     }
     baseUrl = url;
     return baseUrl;
-   
+    
     /*
```

Comment:
What is this commented code block for? If it can be removed, please do so and remove the above empty line along with it.

Reply:
Removed.